packet/bgp.go

```diff
@@ -22,9 +23,16 @@ type bgp struct {
 	k8sclient                                                 kubernetes.Interface
 	peerASN, localASN                                         int
 	annotationLocalASN, annotationPeerASNs, annotationPeerIPs string
+	nodeSelector                                              labels.Selector
 }
 
-func newBGP(client *packngo.Client, project string, localASN, peerASN int, annotationLocalASN, annotationPeerASNs, annotationPeerIPs string) *bgp {
+func newBGP(client *packngo.Client, project string, localASN, peerASN int, annotationLocalASN, annotationPeerASNs, annotationPeerIPs string, nodeSelector string) *bgp {
```

Comment:
I think it is high time this got moved into a parameter struct, or variadic function parameters. But not for this PR; one issue at a time.

Reply:
Created #102 for tracking then.